homie-controller/src/lib.rs

```diff
@@ -637,18 +637,11 @@ mod tests {
 
     fn expect_subscriptions(requests_rx: &Receiver<Request>, subscription_topics: &[&str]) {
         let requests: Vec<_> = (0..subscription_topics.len())
```

Comment:
Just spotted this (I might not have re-reviewed properly after you added this helper in the original PR
you're doing `.map(|_| ...)` below, so this line could be written as:

```suggestion
        let requests: Vec<_> = subscription_topics.iter()
```

Reply:
Done.